lib/serviceDiscovery/engines/consul/index.js

```diff
@@ -60,7 +60,8 @@ class ConsulService extends Service {
 
 		this._consul.session.create({
 			behavior: 'delete',
-			ttl: `${Math.ceil(this.ttl / 1000)}s`
+			ttl: `${Math.ceil(this.ttl / 1000)}s`,
+			lockdelay: '0s'
```

Comment:
Should this be made configurable instead?

Reply:
Lockdelay needs to be 0 in order for the rest to work.  If the lock delay is anything but zero, deleting the session doesn't actually release the lock until the delay has passed.  The intent from Consul's end is to provide lock holders with the opportunity to see their session be invalidated and have a window to fix it, but we're not doing that.